dear_petition/petition/admin.py

```diff
@@ -39,6 +39,33 @@ class CIPRSRecordInline(admin.StackedInline):
     extra = 1
 
 
+class OffenseRecordInline(admin.StackedInline):
+    model = models.OffenseRecord
+    extra = 1
+
+@admin.register(models.Offense)
+class OffenseAdmin(admin.ModelAdmin):
+    list_display = ("pk", "offense_record_count", "ciprs_record", "disposed_on", "disposition_method")
+    list_filter = ("disposed_on",)
+    date_hierarchy = "disposed_on"
+    ordering = ("-disposed_on",)
+    inlines = (OffenseRecordInline,)
+
+    def get_queryset(self, request):
+        queryset = super().get_queryset(request)
+        queryset = queryset.annotate(_offense_record_count=Count("offense_records", distinct=True),)
+        return queryset
+
+    def offense_record_count(self, obj):
+        return obj._record_count
```

Comment:
I'm not sure. I referenced Batch as an example. Do you know for sure? If so, I 'll make a change here and to Batch.

Reply:
Changed back to `offense_record_count`. 

Got this error:
`<class 'dear_petition.petition.admin.BatchAdmin'>: (admin.E108) The value of 'list_display[2]' refers to 'record_count', which is not a callable, an attribute of 'BatchAdmin', or an attribute or method on 'petition.Batch'.
<class 'dear_petition.petition.admin.OffenseAdmin'>: (admin.E108) The value of 'list_display[1]' refers to 'offense_record_count', which is not a callable, an attribute of 'OffenseAdmin', or an attribute or method on 'petition.Offense'.`

Reply:
The elements of `list_display` need to be either fields on the model, or properties of the model, or methods of the admin class.

In the case of `OffenseAdmin`, `list_display` is `["pk", "offense_record_count", "ciprs_record", "disposed_on", "disposition_method"]`, all of which are fields on `Offense`, except for `offense_record_count`, which is a method on `OffenseAdmin`, so that's ok. However, this method (`OffenseAdmin.offense_record_count()`) returns an undefined attribute (`_record_count`) so `offense_record_count` ends up being undefined for admin users. It looked to me like you wanted to return the value of [the new annotated `_offense_record_count` field ](https://github.com/deardurham/dear-petition/blob/ac287ed8ba07e3642720bac759c8e3e5bea3b7ba/dear_petition/petition/admin.py#L63-L65) for the `OffenseAdmin.offense_record_count()` method instead. When I make this change locally, it works for me.

In terms of `BatchAdmin`, it's the same thing: `list_display` need to be either fields on the model, or properties of the model, or methods of the admin class. It looks like `list_display` is `["pk", "label", "record_count", "user", "date_uploaded"]`, all of which are fields on the `Batch` model, except for `record_count`, which you are defining [in line 102](https://github.com/deardurham/dear-petition/blob/ac287ed8ba07e3642720bac759c8e3e5bea3b7ba/dear_petition/petition/admin.py#L102-L103), so that should work. The `BatchAdmin.record_count()` method returns each object's `_record_count`, which is a field [that you annotated on the queryset](https://github.com/deardurham/dear-petition/blob/ac287ed8ba07e3642720bac759c8e3e5bea3b7ba/dear_petition/petition/admin.py#L99), so that should work as well.

Reply:
```suggestion
        return obj._offense_record_count
```

Reply:
To summarize, I'm recommending changing line 69 to be `return obj._offense_record_count`, since `_offense_record_count` exists, and `_record_count` doesn't.